docs/UserGuide.md

```diff
@@ -509,25 +510,25 @@ The constraints of fields are placed in a separate section as we do not wish to
-| **`NUMBER_OF_GUESTS`**  | `ng/`                | Should contain only **numbers**, and it should be at least 1 and at most 4 numbers long.                                                                                                                                                                                                                                                                                               |
-| **`REQUEST`**           | `rq/`                | Is an **optional field**, and it should be at most 500 characters long. It should not contain any prefixes.                                                                                                                                                                                                                                                                            |
-| **`IS_ROOM_CLEAN`**     | `rc/`                | Should contain only `yes`, `no`, `y` or `n` (case-insensitive).                                                                                                                                                                                                                                                                                                                        |
-| **`BILL`**              | `b/`                 | Should contain only **numbers**, and can have up to 2 decimal places.                                                                                                                                                                                                                                                                                                                  |
+| Field                  | Corresponding prefix | Constraint                                                                                                                                                                                                                                                                                                                                                                             |
+|------------------------|----------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **`NAME`**             | `n/`                 | Should contain only **alphabets**, spaces, apostrophes ('), hyphens (-), periods (.) and commas(,). It must start with an alphanumeric character, and it should not be blank.It should not contain any prefixes as well.                                                                                                                                                               |
```

Comment:
I think this part Justin might have updated it as well, so there might be a conflict, just fyi!

Reply:
Will take note!